### DISCUSSION TITLE
Is there an example of using Fluent UI components in an astro application

### DISCUSSION THREAD

**Discussion by nikhilk** [microsoft/fluentui#32942] [Q&A]
I'm trying to use Fluent UI components to build an Astro-based react application. Any pointers on setup or a reference example?

**Comment by Syntarex** [comment]
I will try it today. 
Using the react components seems difficult because of the CSS in JS pattern. They use the react context to pass down configuration. Problem is that Astro's island pattern is not suited for that one. Maybe there is a way to workaround with nanostores but even then I think the downsides and added comlexity is just not healthy. 

I might just use their web components. They are not feature-complete but that way I can build a react component myself if I need the interactivity. 

What I did before was using DaisyUI. You can theme it to look like Fluent2 and stay vanilla/tailwind
